### PR TITLE
Develop rail

### PR DESCRIPTION
**Pull request by peggykri**
the first version of the rules has been created and added

### CODE REVIEW

File: rules.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+- Разрешается уменьшать количество отдыха
+- Разрешается увеличивать рабочую сессию, и, соответственно, Рабочий цикл тоже увеличится. Например: я начал в 8:00, отдых в 8:40, отдохнул до 8:55 и так далее.
+- Уменьшение рабочего цикла считается нарушением
+- Увеличение суммарного отдыха на 30 минут считается нарушением. Например, я начал в 8:00, закончил в 8:30, отдохнул до 8:47 — увеличение суммарного времени отдыха пока составляет 2 минуты.
+- Отдых будет определятся согласно пункту 3
+
+#### Обязательства личного режима
+
+- Играть не более 4(как оценивать?) часов в день
```
Comment (arseniisemenow) [inline rules.md:24]: As I said before, I would just make a schedule with I have no time to play in games

File: rules.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Разрешается увеличивать рабочую сессию, и, соответственно, Рабочий цикл тоже увеличится. Например: я начал в 8:00, отдых в 8:40, отдохнул до 8:55 и так далее.
+- Уменьшение рабочего цикла считается нарушением
+- Увеличение суммарного отдыха на 30 минут считается нарушением. Например, я начал в 8:00, закончил в 8:30, отдохнул до 8:47 — увеличение суммарного времени отдыха пока составляет 2 минуты.
+- Отдых будет определятся согласно пункту 3
+
+#### Обязательства личного режима
+
+- Играть не более 4(как оценивать?) часов в день 
+- Нельзя играть после пробуждения пока не сделаны все базовые утренние дела
+- После 16 ч бодрствования играть запрещено
```
Comment (arseniisemenow) [inline rules.md:26]: Why this two points?

File: rules.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- Играть не более 4(как оценивать?) часов в день 
+- Нельзя играть после пробуждения пока не сделаны все базовые утренние дела
+- После 16 ч бодрствования играть запрещено 
+
+- Если у меня всё сделано, я бодрствую больше 16 ч и есть возможность пойти спать, нужно сделать попытку заснуть, примерно на 30 минут
+
+#### Контроль за питанием
+
+- Готовить самостоятельно каждый день
+- При наличии нормальной еды можно пойти в кафе или столовые только после выполнения учебных обязательств за день (исключая те, что перед сном)
```
Comment (arseniisemenow) [inline rules.md:33]: What is "нормальная еда"?

File: rules.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## 2 Исключения
+
+Пропуск занятия допускается по следующим причинам:
+
+- Болезнь (например, головная боль, высокая температура и тд)
+- Нахождение работы
+- Исключительные случаи, связанные с учёбой в университете (например, необходимость присутствовать до 21:00)
+- Другие явно исключительные случаи
+
+В любой выбранный мной день один раз в неделю разрешается уменьшить объем учебных обязательств в два раза, равномерно по каждому пункту (45 ч в школе -> 225 ч, 15 ч англ -> 075 ч)
```
Comment (arseniisemenow) [inline rules.md:44]: What does mean 45h -> 225h?
15h -> 075h?

File: rules.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+При явно исключительных случаях с расписанием учебных обязательств в школе можно выбрать другое начальное время.
+
+Можно не готовить и поесть в столовых или кафе по следующим причинам:
+
+- Явно никак не получится выделить 1-2 часа на готовку
+- Забыл подготовить продукты, а проснулся уже тогда, когда магазины закрыты
+- Приготовлено недостаточно еды
```
Comment (arseniisemenow) [inline rules.md:52]: Would be better if you write single rule to manage all this rules

File: rules.md (hunk 1 above)
Comment (arseniisemenow) [reply rules.md:24]: So either remove it from here or track this somehow

File: rules.md (hunk 2 above)
Comment (peggykri) [reply rules.md:26]: These two points are very effective for my productivity.

**Pull request closed without merging**